### DISCUSSION TITLE
Miryoku ZMK: caps_word continue-list modification

### DISCUSSION THREAD

**Discussion by kucera-lukas** [manna-harbour/miryoku#190] [Ideas]
It would be great if Miryoku ZMK supported modification of the `caps_word` `continue-list`.

Seems like any changes in `custom_config.h` get overridden in `miryoku_shift_functions.dtsi`.

The quick fix is to just remove the line defining the shift function but introduces the risk of merge conflicts.
```diff
- MIRYOKU_SHIFT_FUNCTION(u_caps_word, &caps_word, &kp CAPS)
```

**Comment by manna-harbour** [comment]
Are you setting the continue list like [this](https://zmk.dev/docs/behaviors/caps-word#continue-list)? Do you have a link to a branch where it doesn't work?

**Reply by kucera-lukas** [reply]
Yes, of course. There isn't any other way to update the default extend list set as far as I'm aware. 

I ended up using urob's excellent smart-word PR - https://github.com/zmkfirmware/zmk/pull/1451 but the original issue is still present. 

I've created 3 branches to demonstrate the issue.

- Base config: https://github.com/kucera-lukas/miryoku_zmk/tree/caps-word-continue-list
- Extension not working: https://github.com/kucera-lukas/miryoku_zmk/tree/caps-word-continue-list-extension
- Extension fixed: https://github.com/kucera-lukas/miryoku_zmk/tree/caps-word-continue-list-extension-fixed (here I didn't bother redefining the whole `u_caps_word` function.
